lib/ohai/plugins/rackspace.rb

```diff
@@ -162,8 +162,8 @@ def get_private_networks
       unless rackspace[:public_ip].nil?
         rackspace[:public_hostname] = begin
           Resolv.getname(rackspace[:public_ip])
-        rescue Resolv::ResolvError, Resolv::ResolvTimeout
-          rackspace[:public_ip]
+                                      rescue Resolv::ResolvError, Resolv::ResolvTimeout
+                                        rackspace[:public_ip]
```

Comment:
Was this formatting change intentional?

Reply:
Yeah rubocop fixed a bug so this is being caught now. They had a regression before that forced the previous formatting